include/tss2/tss2_tpm2_types.h

```diff
@@ -303,12 +303,12 @@ typedef UINT32 TPM2_RC;
 #define TPM2_RC_POLICY             (TPM2_RC_VER1 + 0x026) /* policy failure in math operation or an invalid authPolicy value */
 #define TPM2_RC_PCR                (TPM2_RC_VER1 + 0x027) /* PCR check fail */
 #define TPM2_RC_PCR_CHANGED        (TPM2_RC_VER1 + 0x028) /* PCR have changed since checked. */
-#define TPM2_RC_UPGRADE            (TPM2_RC_VER1 + 0x02D) /* for all commands other than TPM2_FieldUpgradeData this code indicates that the TPM is in field upgrade mode for TPM2_FieldUpgradeData this code indicates that the TPM is not in field upgrade mode */
+#define TPM2_RC_UPGRADE            (TPM2_RC_VER1 + 0x02D) /* for all commands other than TPM2_FieldUpgradeData. This code indicates that the TPM is in field upgrade mode for TPM2_FieldUpgradeData. This code indicates that the TPM is not in field upgrade mode */
```

Comment:
You're chaning the meaning here.

`/* For all commands other than TPM2_FieldUpgradeData this code indicates that the TPM is in field upgrade mode. For TPM2_FieldUpgradeData this code indicates that the TPM is not in field upgrade mode */`

Reply:
OK. The sentence was nonsense. It says that the code means both that it is and is not in field upgrade mode at the same time!

I think the problem is this file was created automatically from a PDF file or some other automated process, so the comments are hard or impossible to understand or are ambiguous. Words are jammed together and periods and dashes are missing.

Thanks for the clarification! Your suggested change makes the comment understandable.

This is the change I made, based on your suggestion:

`/* For all commands, other than TPM2_FieldUpgradeData, this code indicates that the TPM is in field upgrade mode. For TPM2_FieldUpgradeData, this code indicates that the TPM is not in field upgrade mode */`